crates/bevy_reflect/src/lib.rs

```diff
@@ -641,6 +644,16 @@ pub mod __macro_exports {
     impl RegisterForReflection for DynamicArray {}
 
     impl RegisterForReflection for DynamicTuple {}
+
+    /// Stores registration functions of all reflect types that can be automatically registered.
+    #[cfg(not(target_family = "wasm"))]
+    #[allow(non_camel_case_types)]
+    pub struct AUTOMATIC_REFLECT_REGISTRATIONS(pub fn(&mut TypeRegistry));
+    #[cfg(not(target_family = "wasm"))]
+    inventory::collect!(AUTOMATIC_REFLECT_REGISTRATIONS);
```

Comment:
Could this be put in a single block to avoid repeating the `cfg`, which makes it quite unreadable?
```suggestion
    #[cfg(not(target_family = "wasm"))]
    {
        #[allow(non_camel_case_types)]
        pub struct AUTOMATIC_REFLECT_REGISTRATIONS(pub fn(&mut TypeRegistry));
        inventory::collect!(AUTOMATIC_REFLECT_REGISTRATIONS);
    }
```

Reply:
I think the only way to group these top-level statement is to put them in a module. This could work I guess, although I'm not sure how much clearer it will be as it will add to the already long path to `AUTOMATIC_REFLECT_REGISTRATIONS`.

Reply:
Ah right, somehow I tought this was inside a function. A module doesn't seem any clearer than this, so I guess it's better to leave this code like this. Feel free to resolve the comment.